src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -141,8 +141,10 @@ public override async Task<bool> Execute()
                         if (!await this.PromptPackageIdentifierAndCheckDuplicates(manifests))
                         {
                             Console.WriteLine();
-                            Logger.ErrorLocalized(nameof(Resources.PackageIdAlreadyExists_Error));
-                            return false;
+                            if (!Prompt.Confirm(Resources.PackageIdAlreadyExists_Message))

```

Comment:
What's the rationale for making this non-blocking?

Reply:
Kevin did not like that he could not create a manifest that already existed. I think this is the best compromise to notify the user that their package is a duplicate while still allowing them to use our tool to generate any manifest they want

Reply:
As discussed offline with @KevinLaMS, we shouldn't allow users to bypass this. For testing purposes, we can always manipulate the string to avoid a duplicate

Reply:
Reverted changes

Reply:
Reverted changes.